packages/headless/typedoc/assets/docs-style.css

```diff
@@ -0,0 +1,67 @@
+:root {
+  --light-color-background: #fff;
+  --light-color-background-secondary: #333357;
+  --light-color-text: #282829;
+  --light-color-text-aside: #399ffe;
+  --dark-color-background: #3f3f71;
+  --dark-color-background-secondary: #333357;
+  --dark-color-text: #f1f2ff;
+  --dark-color-text-aside: #399ffe;
+}
+
+body {
+  font:
+    100 1em/1.625 'canada-type-gibson',
+    sans-serif;
+}
+
+.tsd-breadcrumb a {
+  font-weight: 400;
+}
+
+@media (max-width: 768px) {
+  #tsd-search {
+    display: none;
+  }
+}
+
+#tsd-search {
```

Comment:
On the docs site, the search box is resized at different breakpoints, ideally we'd do the same here.